### PR TITLE
Task-Backend Users

### PR DESCRIPTION
**Pull request by mirantrix**
• Ajuste del componente Users
• Agregar elementos a la lista

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
 
 class App extends Component {
+
   render() {
     return (
       <div className="App">
-        <header className="App-header">
-          <Users/>
-        </header>
+        <Users/>
```
Comment (jalmarazg) [inline src/App.js:11]: Maybe we could use something more descriptive like `UserList`

File: src/components/User/user.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  }
+
+  render() {
+    return (
+      <div>
+        <h2>MoneyJump</h2>
+        <ul>
+          <li key={this.state.user.id}>
+            <p>{this.state.user.name}</p>
+            <img src={this.state.user.image} alt="Logo"/>
```
Comment (jalmarazg) [inline src/components/User/user.js:25]: We could the JS concept of Object Destructuring to have a cleaner code 

Reference: [ES6 Destructuring: The Complete Guide](https://codeburst.io/es6-destructuring-the-complete-guide-7f842d08b98f)

File: src/App.js (hunk 1 above)
Comment (mirantrix) [reply src/App.js:11]: Got it

File: src/components/User/user.js (hunk 2 above)
Comment (mirantrix) [reply src/components/User/user.js:25]: ok, thanks for sharing the resource, I will read it and try to apply it

**Pull request closed without merging**